interface/usergroup/addrbook_edit.php

```diff
@@ -273,241 +265,291 @@ function typeSelect(a) {
+		<input type='text' size='10' name='form_city' maxlength='30' value='<?php echo attr($row['city']); ?>' class='form-control form-control-sm inputtext' placeholder="<?php echo xlt('City'); ?>" />
+	</div>
+	<div class="col-2">
+		<label for="form_state" class="font-weight-bold col-form-label col-form-label-sm"><?php echo xlt('State')."/".xlt('county'); ?>:</label>
+	</div>
+	<div class="col">
+		<input type='text' size='10' name='form_state' maxlength='30' value='<?php echo attr($row['state']); ?>' class='form-control form-control-sm inputtext' placeholder="<?php echo xlt('State')."/".xlt('county'); ?>" />
+	</div>
+	<div class="col-2">
+		<label for="form_zip" class="font-weight-bold col-form-label col-form-label-sm"><?php echo xlt('Postal code'); ?>:</label>
+	</div>
+	<div class="col">
+		<input type='text' size='10' name='form_zip' maxlength='20' value='<?php echo attr($row['zip']); ?>' class='form-control form-control-sm inputtext' placeholder="<?php echo xlt('Postal code'); ?>" />
+	</div>
+</div>
+
+<div class="form-row my-1 align-items-center">
+	<div class="col-2">
+		<label for="form_street2 form_streetb2" class="font-weight-bold col-form-label col-form-label-sm"><?php echo xlt('Alt Address'); ?>:</label>
+	</div>
+	<div class="col-10">
+		<input type='text' size='40' name='form_street2' maxlength='60' value='<?php echo attr($row['street2']); ?>' class='form-control form-control-sm mb-1 inputtext' placeholder="<?php echo xlt('Address Line 1'); ?>" />
+		<input type='text' size='40' name='form_streetb2' maxlength='60' value='<?php echo attr($row['streetb2']); ?>' class='form-control form-control-sm mt-1 inputtext' placeholder="<?php echo xlt('Address Line 2'); ?>" />
+	</div>
+</div>
+
+<div class="form-row my-1">
+	<div class="col-2">
+		<label for="form_city2" class="font-weight-bold col-form-label col-form-label-sm"><?php echo xlt('Alt City'); ?>:</label>
+	</div>
+	<div class="col-auto">
+		<input type='text' size='10' name='form_city2' maxlength='30' value='<?php echo attr($row['city2']); ?>' class='form-control form-control-sm inputtext' placeholder="<?php echo xlt('Alt City'); ?>" />
+	</div>
+	<div class="col-auto">
+		<label for="form_state2" class="font-weight-bold col-form-label col-form-label-sm"><?php echo xlt('Alt State')."/".xlt('county'); ?>:</label>
+	</div>
+	<div class="col-auto">
+		<input type='text' size='10' name='form_state2' maxlength='30' value='<?php echo attr($row['state2']); ?>' class='form-control form-control-sm inputtext' placeholder="<?php echo xlt('Alt State')."/".xlt('county'); ?>" />
+	</div>
+	<div class="col-auto">
+		<label for="form_zip2" class="font-weight-bold col-form-label col-form-label-sm"><?php echo xlt('Alt Postal code'); ?>:</label>
+	</div>
+	<div class="col-auto">
+		<input type='text' size='10' name='form_zip2' maxlength='20' value='<?php echo attr($row['zip2']); ?>' class='form-control form-control-sm inputtext' placeholder="<?php echo xlt('Alt Postal code'); ?>" />
```

Comment:
All the placeholder translations should use xla and not xlt (in order to escape properly within html attributes).